BUILD.md

```diff
@@ -478,13 +478,13 @@ poetry shell
 Install [.NET SDK](https://dotnet.microsoft.com/en-us/download) version 6 or later. Then, open a terminal and install the latest version of [WiX Toolset .NET tool](https://wixtoolset.org/) **v5** with:
 
 ```sh
-dotnet tool install --global wix --version 5.*
+dotnet tool install --global wix --version 5.0.2
```

Comment:
The more I think about it, the more I start the believe this should just be a script that we provide, that should run on windows, rather than a text that we copy/paste.

Probably not for this PR though.

Reply:
Noted. I see that Microsoft is offering manual instructions as well, so I guess both things would work. Punting it for a later time.